controller.go

```diff
@@ -396,7 +399,7 @@ func (c *Controller) syncHandler(key string) error {
 
 			provisioner, _ := provision.NewDigitalOceanProvisioner(c.infraConfig.GetAccessKey())
 
-			userData := makeUserdata(tunnel.Spec.AuthToken)
+			userData := makeUserdata(tunnel.Spec.AuthToken, true)
```

Comment:
What does `true` do? If it's adding sudo, how about we add a const for it for readability i.e.:

```
const withSudo true
const withoutSudo false
```